snapcraft.yaml

```diff
@@ -73,16 +74,23 @@ parts:
       - --buildtype=release
     build-packages:
       - libgnome-autoar-0-dev
+      - libsoup2.4-dev
+      - libsoup-gnome2.4-dev
+      - librest-dev
+      - libgoa-1.0-dev
     stage-packages:
       - libgnome-autoar-0-0
+      - libsoup-2.4-1
+      - libsoup-gnome-2.4-1
+      - librest-1.0-0
+      - libgoa-1.0-0b
     build-snaps:
       - gnome-46-2404-sdk/latest/candidate
 
-
   cleanup:
     after: [ gnome-recipes ]
     plugin: nil
-    build-snaps: [core24, gtk-common-themes, gnome-46-2404]
+    build-snaps: [core24, gtk-common-themes, gnome-46-2404/latest/candidate]
```

Comment:
I don't think we should hardcode the candidate there. Is there anything that is going to break if we build again stable of just the cleanup isn't going to be optimal? If there is a breakage we should imho just wait for gnome-46-2404 candidate to promote to stable, otherwise we can merge already and we will do another rebuild after promotion...

Reply:
The problem is that, until the current candidate goes into stable, it will take the current gnome-46-2404 stable, where libsoup-2.4 still exists, and remove it from the snap, so it won't work. So either we publish this now and remove that "/latest/candidate" part when the current gnome-46-2404 candidate goes stable, or we remove that now and wait until gnome-46-2404 candidate goes stable to publish it.

Reply:
alright